woocommerce/woocommerce-android
# Set payment description

Parent issue #4039 

Merge instrucitons
1. Make sure https://github.com/woocommerce/woocommerce-android/pull/4106 is merged
2. Remove "do not merge" label
3. Merge this PR

Sets payment description to each card present payment. The description is visible in Stripe's dashboard next to the transaction.

To test
- use a store with WCPay in test mode and orders in US $
1. Open detail of an order
2. Tap on "Collect payment"
3. Walk through the connection and payment flows
4. Open Stripe's dashboard (if you don't have access to the dashboard, just put a breakpoint [here](https://github.com/woocommerce/woocommerce-android/blob/c696819e9a7b74b09dc8e8871d79a58c9d3bf177/libs/cardreader/src/debug/java/com/woocommerce/android/cardreader/internal/payments/actions/CreatePaymentAction.kt#L59) and verify the description is set)
5. Verify the transaction has description set

Update release notes:
